components/Item/UpNext.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div class="container">
+    <div class="d-flex flex-column flex-grow-1">
+      <h2 class="countdownHeader">
+        {{ $t('nextEpisodePlayingIn') }}
+        <span class="primary--text darken-2"
+          >{{ timeLeft }} {{ $t('seconds') }}</span
+        >
+      </h2>
+      <h3 class="title subtitle-1">
+        {{ nextSeriesName }} - S{{ nextSeasonNumber }}:E{{
+          nextEpisodeNumber
+        }}
+        - {{ nextName }}
+      </h3>
```

Comment:
This should be translated. See [here](https://github.com/jellyfin/jellyfin-vue/blob/master/components/Item/Card/Card.vue#L222-L225) and [here](https://github.com/jellyfin/jellyfin-vue/blob/master/locales/en-US.json#L364)

Reply:
Ah, neat. I'm assuming you meant to link to line 344 and not `"version": "Version",` in the second link?

Reply:
Yes